pynitrokey/nk3/otp_app.py

```diff
@@ -243,3 +265,101 @@ def verify_code(self, cred_id: bytes, code: int) -> bool:
         ]
         res = self._send_receive(Instruction.VerifyCode, structure=structure)
         return res.hex() == "7700"
+
+    def set_code_cli(self, passphrase: str) -> None:
```

Comment:
nit: I’d prefer using `set_code` for the high-level function and something like `set_code_raw` for the low-level function (same for `validate`).

Reply:
Makes sense. Done.
In general here are two abstraction levels mixed, but I think it  makes little sense to add high level wrapper for just 2 functions.